### PR TITLE
Install Apache Solr Operator Helm Chart Repository and CRDs to prod

### LINKED ISSUE #110: Smart Village: Install Apache Solr Operator Helm Chart Repository and CRDs
The [Smart Village Collaboratory project](https://research.redhat.com/blog/research_project/creating-a-global-open-research-platform-to-better-understand-social-sustainability-using-data-from-a-real-life-smart-village/) would like to install the Apache Solr Operator and CRDs to NERC. There are two important parts here. The Apache Zookeeper cluster for running Solr as a clustered cloud, as well as our smart village application which is a Vert.x application that is also clustered between pods with the Apache Zookeeper cluster. The second part is Apache Solr which is the search engine for the Smart Village application. The search engine powers the API for extremely fast querying, pagination, filtering, faceting, pivoting, sorting and more. This allows for building custom analytics and dashboards that are extremely powerful. You can find the [CRDs for Apache Solr](https://solr.apache.org/operator/downloads/crds/v0.6.0/all-with-dependencies.yaml) here, and the [Helm Chart Repository for Apache Solr](https://github.com/computate-org/smartabyar-smartvillage/blob/main/openshift/kustomize/bundles/solr/base/helmchartrepositories/solr-helm-repo/helmchartrepository.yaml) here.

### PR DESCRIPTION
**Pull request by computate**
The [Smart Village Collaboratory project](https://research.redhat.com/blog/research_project/creating-a-global-open-research-platform-to-better-understand-social-sustainability-using-data-from-a-real-life-smart-village/) would like to install the Apache Solr Operator and CRDs to NERC. There are two important parts here. The Apache Zookeeper cluster for running Solr as a clustered cloud, as well as our smart village application which is a Vert.x application that is also clustered between pods with the Apache Zookeeper cluster. The second part is Apache Solr which is the search engine for the Smart Village application. The search engine powers the API for extremely fast querying, pagination, filtering, faceting, pivoting, sorting and more. This allows for building custom analytics and dashboards that are extremely powerful. You can find the [CRDs for Apache Solr](https://solr.apache.org/operator/downloads/crds/v0.6.0/all-with-dependencies.yaml) here, and the [Helm Chart Repository for Apache Solr](https://github.com/computate-org/smartabyar-smartvillage/blob/main/openshift/kustomize/bundles/solr/base/helmchartrepositories/solr-helm-repo/helmchartrepository.yaml) here. 

Close https://github.com/OCP-on-NERC/operations/issues/110

### CODE REVIEW

**Review by jtriley — COMMENTED** [review]
LGTM. @larsks Would you prefer to have these CRDs in base instead (e.g. `cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions`)?

**Review by aabaris — APPROVED** [review]

File: cluster-scope/bundles/solr-helm-repo/solrbackups.yaml
Comment (larsks) [inline cluster-scope/bundles/solr-helm-repo/solrbackups.yaml:1]: Subdirectories of the `bundles` directory should only contain `kustomization.yaml` files.

**Review by larsks — APPROVED** [review]

**Review by larsks — CHANGES_REQUESTED** [review]
Sorry, previous review should have been "request changes".

File: cluster-scope/bundles/solr-helm-repo/solrbackups.yaml
Comment (computate) [reply cluster-scope/bundles/solr-helm-repo/solrbackups.yaml:1]: Sure @larsks , I moved the CRDs to `cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/solr-helm-repo`

**Review by larsks — APPROVED** [review]

**Review by jtriley — APPROVED** [review]

**Pull request merged**